ios/FluentUI/TwoLineTitleView/TwoLineTitleView.swift

```diff
@@ -204,38 +223,34 @@ open class TwoLineTitleView: UIView, TokenizedControlInternal {
 
         applyStyle()
 
-        titleButton.addTarget(self, action: #selector(onTitleButtonHighlighted), for: [.touchDown, .touchDragInside, .touchDragEnter])
-        titleButton.addTarget(self, action: #selector(onTitleButtonUnhighlighted), for: [.touchUpInside, .touchDragOutside, .touchDragExit])
-        titleButton.addTarget(self, action: #selector(onTitleButtonTapped), for: [.touchUpInside])
-        addSubview(titleButton)
+        addGestureRecognizer(UITapGestureRecognizer(target: self, action: #selector(onTitleTapped)))
 
-        titleButton.addSubview(titleButtonLeadingImageView)
-        titleButton.addSubview(titleButtonLabel)
-        titleButton.addSubview(titleButtonTrailingImageView)
+        // We do all of this instead of a simple contain(view:) to account for the minimum touch size
+        addSubview(containingStackView)
+        containingStackView.translatesAutoresizingMaskIntoConstraints = false
 
-        subtitleButton.addTarget(self, action: #selector(onSubtitleButtonHighlighted), for: [.touchDown, .touchDragInside, .touchDragEnter])
-        subtitleButton.addTarget(self, action: #selector(onSubtitleButtonUnhighlighted), for: [.touchUpInside, .touchDragOutside, .touchDragExit])
-        subtitleButton.addTarget(self, action: #selector(onTitleButtonTapped), for: [.touchUpInside])
-        addSubview(subtitleButton)
+        widthAnchor.constraint(greaterThanOrEqualToConstant: TokenSetType.minimumTouchSize.width).isActive = true
+        heightAnchor.constraint(greaterThanOrEqualToConstant: TokenSetType.minimumTouchSize.height).isActive = true
 
-        subtitleButton.addSubview(subtitleButtonLabel)
-        subtitleButton.addSubview(subtitleButtonImageView)
+        centerXAnchor.constraint(equalTo: containingStackView.centerXAnchor).isActive = true
+        centerYAnchor.constraint(equalTo: containingStackView.centerYAnchor).isActive = true
+        widthAnchor.constraint(greaterThanOrEqualTo: containingStackView.widthAnchor).isActive = true
+        heightAnchor.constraint(greaterThanOrEqualTo: containingStackView.heightAnchor).isActive = true
```

Comment:
Would it be possible to group all these constraints in one `NSLayoutConstraint.activate([])`?

Reply:
We could, but it'd make the library size grow by 1,992 bytes. Is there any documentation of a large performance benefit from a single `NSLayoutConstraint.activate`?

Reply:
I don't know if there's a large performance benefit, but it is the style most of the other files in the repo follow. Considering the binary hit, though, sounds like something we should reconsider.

Reply:
Opened https://github.com/microsoft/fluentui-apple/issues/1741 to further discuss this. In the meantime, I think we can resolve this thread.

Reply:
https://developer.apple.com/documentation/uikit/nslayoutconstraint/1526955-activate#:~:text=This%20convenience%20method%20provides%20an%20easy%20way%20to%20activate%20a%20set%20of%20constraints%20with%20one%20call.%20The%20effect%20of%20this%20method%20is%20the%20same%20as%20setting%20the%20isActive%20property%20of%20each%20constraint%20to%20true.%20Typically%2C%20using%20this%20method%20is%20more%20efficient%20than%20activating%20each%20constraint%20individually.